addons/account_asset/models/account_asset.py

```diff
@@ -15,9 +16,9 @@ class AccountAssetCategory(models.Model):
     active = fields.Boolean(default=True)
     name = fields.Char(required=True, index=True, string="Asset Type")
     account_analytic_id = fields.Many2one('account.analytic.account', string='Analytic Account')
-    account_asset_id = fields.Many2one('account.account', string='Asset Account', required=True, domain=[('internal_type','=','other'), ('deprecated', '=', False)], help="Account used to record the purchase of the asset at its original price.")
-    account_depreciation_id = fields.Many2one('account.account', string='Depreciation Entries: Asset Account', required=True, domain=[('internal_type','=','other'), ('deprecated', '=', False)], help="Account used in the depreciation entries, to decrease the asset value.")
-    account_depreciation_expense_id = fields.Many2one('account.account', string='Depreciation Entries: Expense Account', required=True, domain=[('internal_type','=','other'), ('deprecated', '=', False)], oldname='account_income_recognition_id', help="Account used in the periodical entries, to record a part of the asset as expense.")
+    account_asset_id = fields.Many2one('account.account', string='Asset Account', required=True, domain=[('internal_type', '=', 'other'), ('deprecated', '=', False)], help="Account used to record the purchase of the asset at its original price.")
```

Comment:
While we are at it, we could use a multi line definition. We do not limitate ourselves to 80 characters, but trying to stick on 100 or 120 could be interesting.